serverless/common/openvino/model_loader.py

```diff
@@ -57,16 +50,23 @@ def infer(self, image, preprocessing=True):
         inputs = {self._input_blob_name: image}
         if self._input_info_name:
             inputs[self._input_info_name] = [h, w, 1]
+        return inputs
 
+    def infer(self, image, preprocessing=True):
+        inputs = self._prepare_inputs(image, preprocessing)
         results = self._net.infer(inputs)
         if len(results) == 1:
             return results[self._output_blob_name].copy()
         else:
             return results.copy()
 
+    def async_infer(self, image, preprocessing=True, request_id=0):
+        inputs = self._prepare_inputs(image, preprocessing)
+        return self._net.start_async(request_id=request_id, inputs=inputs)
+
     def input_size(self):
         return self._input_layout[2:]
 
     @property
     def layers(self):
-        return self._network.layers
+        return self._network.layers
```

Comment:
@nmanovic @bsekachev btw just for your information, layers property of IENetwork was deprecated and removed in 2021.2, so maybe it makes sense to get rid of it on common ModelLoader and handle it locally in yolo function (the only one place where i found a usage)

Reply:
Here is an example of auto annotated image. age-gender looks pretty inaccurate, maybe to keep only emotions-recognition as a showcase?
<img width="1904" alt="Screenshot 2021-12-21 at 18 46 21" src="https://user-images.githubusercontent.com/56150992/146958627-34c5510f-945a-40b5-a315-260ba294d754.png">

Reply:
@mikhail-treskin , I agree. I'm OK to upgrade the OpenVINO version and modify the code.